scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -1054,6 +1055,38 @@ def _prepare_to_stop(self):
         finally:
             self.__condition_var.release()
 
+    def mark_not_new(self, container_id):
```

Comment:
I think `new` is too overloaded in this context.  Can we call this `mark_as_used` where we define if it we've used this warmed cache entry or not.